example/src/screens/InitialScreen.tsx

```diff
@@ -3,19 +3,24 @@ import { Typo } from '@components/Typo';
 import { CardButton } from '@components/buttons/CardButton';
 import * as Application from 'expo-application';
 import React from 'react';
-import { View, StyleSheet, Image } from 'react-native';
+import { View, StyleSheet, Image, Pressable } from 'react-native';
 import { SafeAreaView } from 'react-native-safe-area-context';
+import { useVideoroomState } from 'src/VideoroomContext';
 
 export const InitialScreen = ({ navigation }) => {
+  const { setIsDevMode, isDevMode } = useVideoroomState();
+
   return (
     <BackgroundAnimation>
       <SafeAreaView style={{ flex: 1 }} edges={['bottom']}>
         <View style={styles.content}>
           <View style={styles.header}>
-            <Image
-              style={styles.logo}
-              source={require('@assets/images/Logo.png')}
-            />
+            <Pressable onLongPress={() => setIsDevMode(!isDevMode)}>
```

Comment:
can we have a confirmation modal or something? the way it is right now you might not notice that you're in dev mode

Reply:
Done.